_posts/2020-01-30-database-migration-spring-boot-flyway.md

```diff
@@ -0,0 +1,133 @@
+---
+title: One-Stop Guide to Database Migration with Spring Boot and Flyway
+categories: [spring-boot]
+date: 2020-01-30 05:00:00 +1100
```

Comment:
I replaced the variables you set in the header so that the preview works.

Reply:
👍